src/quicreach.cpp

```diff
@@ -290,28 +289,25 @@ struct ReachConnection : public MsQuicConnection {
                 '\0'};
             QUIC_ADDR_STR AddrStr;
             QuicAddrToString(&RemoteAddr.SockAddr, &AddrStr);
-            unique_lock<mutex> lock(Results.Mutex);
-            printf("%30s   %3u.%03u ms   %3u.%03u ms   %3u.%03u ms   %u:%u %u:%u (%2.1fx)  %4u   %4u     %s   %20s   %s\n",
-                HostName,
-                Stats.Rtt / 1000, Stats.Rtt % 1000,
-                InitialTime / 1000, InitialTime % 1000,
-                HandshakeTime / 1000, HandshakeTime % 1000,
-                (uint32_t)Stats.SendTotalPackets,
-                (uint32_t)Stats.RecvTotalPackets,
-                (uint32_t)Stats.SendTotalBytes,
-                (uint32_t)Stats.RecvTotalBytes,
-                Amplification,
-                Stats.HandshakeClientFlight1Bytes,
-                Stats.HandshakeServerFlight1Bytes,
-                Version == QUIC_VERSION_1 ? "v1" : "v2",
-                AddrStr.Address,
-                HandshakeTags);
+            std::unique_lock<std::mutex> lock(Results.Mutex);
+            std::cout << std::setw(30) << HostName << "   "
```

Comment:
I think this conversion from `printf` to `cout` really makes it harder to read and understand what is being printed out.

Reply:
Hello @nibanks, It will be printed Hostname taking 30 char padding.